TableGenerator.Tests/DomainTests.fs

```diff
@@ -13,21 +13,24 @@ open System
 let ``it can Shorten master branch Name``() =
     let branch =
         { GitBranchName = "master"
-          DisplayName = "Master<br>(5.0.x&nbsp;Runtime)" }
+          DisplayName = "Master<br>(5.0.x&nbsp;Runtime)"
+          AkaMsChannel = Some("net5/dev") }
```

Comment:
do we have a convention for the channel? so we don't have to ask Matt whenever there is a branch.